src/e2e-test/java/io/cdap/plugin/utils/CdfPluginPropertyLocator.java

```diff
@@ -46,35 +51,39 @@ public enum CdfPluginPropertyLocator {
-      .put("objectsToCreate", CdfPluginPropertyLocator.GCS_CREATE_OBJECTS_TO_CREATE)
-      .put("createFailIfObjectExists", CdfPluginPropertyLocator.GCS_CREATE_FAIL_IF_OBJECT_EXISTS)
-      .put("gcsMoveSourcePath", CdfPluginPropertyLocator.GCS_MOVE_SOURCE_PATH)
-      .put("gcsMoveDestinationPath", CdfPluginPropertyLocator.GCS_MOVE_DESTINATION_PATH)
-      .build();
+            .put("projectId", CdfPluginPropertyLocator.PROJECT_ID)
+            .put("datasetProjectId", CdfPluginPropertyLocator.DATASET_PROJECT_ID)
+            .put("dataset", CdfPluginPropertyLocator.DATASET)
+            .put("table", CdfPluginPropertyLocator.TABLE)
+            .put("format", CdfPluginPropertyLocator.FORMAT)
+            .put("path", CdfPluginPropertyLocator.PATH)
+            .put("sampleSize", CdfPluginPropertyLocator.SAMPLE_SIZE)
+            .put("delimiter", CdfPluginPropertyLocator.DELIMITER)
+            .put("skipHeader", CdfPluginPropertyLocator.SKIP_HEADER)
+            .put("pathSuffix", CdfPluginPropertyLocator.SUFFIX)
+            .put("encryptionKeyName", CdfPluginPropertyLocator.CMEK_KEY)
+            .put("serviceAccountType", CdfPluginPropertyLocator.SERVICE_ACCOUNT_TYPE)
+            .put("serviceAccountFilePath", CdfPluginPropertyLocator.SERVICE_ACCOUNT_PATH)
+            .put("serviceAccountJSON", CdfPluginPropertyLocator.SERVICE_ACCOUNT_JSON)
+            .put("truncateTable", CdfPluginPropertyLocator.TRUNCATE_TABLE)
+            .put("updateTableSchema", CdfPluginPropertyLocator.UPDATE_TABLE_SCHEMA)
+            .put("topic", CdfPluginPropertyLocator.PUBSUB_TOPIC)
+            .put("maximumBatchCount", CdfPluginPropertyLocator.PUBSUB_MAXIMUM_BATCH_COUNT)
+            .put("maximumBatchSize", CdfPluginPropertyLocator.PUBSUB_MAXIMUM_BATCH_SIZE)
+            .put("publishDelayThreshold", CdfPluginPropertyLocator.PUBSUB_PUBLISH_DELAY_THRESHOLD)
+            .put("retryTimeout", CdfPluginPropertyLocator.PUBSUB_RETRY_TIMEOUT)
+            .put("errorThreshold", CdfPluginPropertyLocator.PUBSUB_ERROR_THRESHOLD)
+            .put("outputSchema", CdfPluginPropertyLocator.OUTPUT_SCHEMA_MACRO_INPUT)
+            .put("objectsToDelete", CdfPluginPropertyLocator.GCS_DELETE_OBJECTS_TO_DELETE)
+            .put("objectsToCreate", CdfPluginPropertyLocator.GCS_CREATE_OBJECTS_TO_CREATE)
+            .put("createFailIfObjectExists", CdfPluginPropertyLocator.GCS_CREATE_FAIL_IF_OBJECT_EXISTS)
+            .put("gcsMoveSourcePath", CdfPluginPropertyLocator.GCS_MOVE_SOURCE_PATH)
+            .put("gcsMoveDestinationPath", CdfPluginPropertyLocator.GCS_MOVE_DESTINATION_PATH)
+            .put("partitionFrom", CdfPluginPropertyLocator.PARTITION_START_DATE)
+            .put("partitionTo", CdfPluginPropertyLocator.PARTITION_END_DATE)
+            .put("filter", CdfPluginPropertyLocator.FILTER)
+            .put("Output Schema-macro-input", CdfPluginPropertyLocator.OUTPUT_SCHEMA)
+            .build();
```

Comment:
why this change?

Reply:
I was unable to add my steps properly in this field because the alignments were incorrect. To fix this, I copied and pasted the same content and added my steps without making any other changes.

Reply:
please fix the code style in your editor.

Reply:
Refactor CdfPluginPropertyLocator alignment and update build dependencies.
-Fixed indentation alignment for partitionFrom, partitionTo, filter and Output Schema.
-Updated pom.xml as it is.